Token.js

```diff
@@ -108,6 +108,7 @@ class Token {
 	}
 	rotate(newRotation) {
 		if (this.options.locked) return; // don't allow rotation if the token is locked
+		if (this.options.restrictPlayerMove) return; // don't allow rotation if the token is locked
```

Comment:
I think this should be if (!window.DM && this.options.restrictPlayerMove ) 

so this block only applies to non DM